packs/data/bestiary-family-ability-glossary.db/vampire-nosferatu-dominate.json

```diff
@@ -13,7 +13,7 @@
             "value": 2
         },
         "description": {
-            "value": "<p>The nosferatu can cast <em>@UUID[Compendium.pf2e.spells-srd.Dominate]{Dominate}</em> at will as a divine innate spell. Casting it requires staring into the target's eyes, giving the spell the visual trait (@Check[type:will|dc:]). The save DC uses a high DC for the nosferatu's level, and a creature that succeeds is temporarily immune to that nosferatu's Dominate for 24 hours. Fully destroying the nosferatu ends the domination, but merely reducing the nosferatu to 0 HP is insufficient to break the spell.</p>"
```

Comment:
Why remove?

Reply:
Oops, now I see - that's the save for the spell effect. Added it again and also added it for the true vampire dominate ability.